docs/content/en/docs/reference/metrics.md

```diff
@@ -0,0 +1,420 @@
+---
+title: "Metrics"
+description: >
+  This reference is autogenerated from the Tetragon Prometheus metrics registry
+  using github.com/isovalent/metricstool
```

Comment:
This looks a bit off since links are not handled in the description and that's referring to `isovalent` repo. Maybe we can just remove "using github.com/isovalent/metricstool", or not if you want to promote that one, but then maybe in the body?

It looks a bit weird in the section TOC as well.

<img width="1249" alt="image" src="https://github.com/cilium/tetragon/assets/11256051/6533e7f8-716a-4612-b2f7-0b2c0480960e">

Reply:
Yeah, this looks weird, I'll just remove it.